addons/mail/static/src/messaging/component/activity/activity.js

```diff
@@ -121,9 +121,9 @@ class Activity extends Component {
      * @private
      * @param {MouseEvent} ev
      */
-    _onClickCancel(ev) {
+    async _onClickCancel(ev) {
         ev.preventDefault();
-        this.activity.delete();
+        await this.activity.deleteRecord();
```

Comment:
I don't think we care to await here as the browser is not going to care if the handlers are async. cc @alexkuhn just to be sure

Reply:
let's avoid async code as much as possible in components, so avoid `await` there as much as possible